src/Lfm.php

```diff
@@ -191,8 +191,18 @@ public function allowShareFolder()
      */
     public function translateFromUtf8($input)
     {
+        $rInput = [];
+
         if ($this->isRunningOnWindows()) {
-            $input = iconv('UTF-8', mb_detect_encoding($input), $input);
+            if (is_array($input)) {
+                foreach ($input as $k => $i) {
+                    $rInput[] = iconv('UTF-8', mb_detect_encoding($i), $i);
+                }
+            } else {
+                $rInput = $input;
+            }
+        } else {
+            $rInput = $input;
         }
 
         return $input;
```

Comment:
$input or $rInput?

Reply:
i think this is better code:
public function translateFromUtf8($input)
{
    if ($input == null)
        return $input;
    if ($this->isRunningOnWindows()) {
        if (is_array($input)) {
            for($i = 0 ; $i < count($input);$i++)
                $input[$i] = iconv('UTF-8', mb_detect_encoding($input[$i]), $input[$i]);

        } else
            $input = iconv('UTF-8', mb_detect_encoding($input), $input);
    }

    return $input;
}